pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -2675,7 +2676,14 @@ public MessageId terminate(
             @ApiParam(value = "Is authentication required to perform this operation")
             @QueryParam("authoritative") @DefaultValue("false") boolean authoritative) {
         validatePersistentTopicName(tenant, namespace, encodedTopic);
```

Comment:
need catch exception

Reply:
I think we don't need to. The exception it throws is of type RestException.